native/src/snappyclient/cpp/ColumnDescriptor.cpp

```diff
@@ -191,6 +191,6 @@ uint32_t ColumnDescriptor::getDisplaySize() const noexcept {
       return (size > 0 ? size : 30);
     default:
       size = getPrecision();
-      return (size > 0 ? size : 15);
+      return (size > 0 ? (size + 2) : 15);
```

Comment:
Can this potentially affect any other type that might be getting covered in default case and (not just decimal and numeric)?

Reply:
The added case for Decimal type. As per Microsoft document, we should add + 2 for the decimal type. Revert back the changes for default case